### DISCUSSION TITLE
`Popover` `onPointerDownOutside` doesn't seem to work as expected

### DISCUSSION THREAD

**Discussion by DonHuskini** [radix-ui/primitives#1867] [Help]
I have a card with nested click events which I don't want to bubble up to parent nodes.

When I click outside of the Popover I expect it to close it and stop the propagation of the event however it triggers the onClick event of the parent container. Example: https://stackblitz.com/edit/react-ts-7tupkg?file=App.tsx

**Comment by benoitgrelard** [comment]
The reason this doesn't work as you expect is because `onPointerDownOutside` is using a custom event which itself is derived from a pointer event on the document level, so stopping propagation won't do what you expect.

**Reply by joelharkes** [reply]
@benoitgrelard  is there actually a way avoid to avoid propagation of the click?
